common/translation_utils/make_pos.py

```diff
@@ -0,0 +1,136 @@
+import polib
+import csv
+import copy
+import os
+
+class Translation(object):
+    pass
+
+def make_pos_run():
+
+    def make_translation():
+            translation = Translation()
+            po = polib.POFile()
+            po.metadata = copy.deepcopy(po_metadata_template)
+            translation.pofile = po
+            return translation
+
+    po_metadata_template = {
```

Comment:
This is also used in `make_testing_pos`. Maybe factor it out into a separate file and create a `get_po_metadata_template` function that returns a template. Then import and use that function for both of these scripts.

(Note: The reason to wrap it into a function is that you get a new instance of the template each time, rather than sharing the same instance across multiple scripts.)

Reply:
Have factored out into a separate file as advised